test/jackdaw/streams_test.clj

```diff
@@ -725,8 +749,8 @@
 
           ;; if we don't set the `grace` period of the `TimeWindows`, the
           ;; default is used: 24h - window
-          window       (Duration/ofMillis 100)
-          grace        (Duration/ofMillis 1)
+          window (Duration/ofMillis 100)
```

Comment:
Would prefer not to realign unrelated code.

Reply:
🤦🏻

Reply:
Fixed